legacy/securityconfig_v6/audit.yml

```diff
@@ -0,0 +1,45 @@
+config:
```

Comment:
Is there a plan to backport the functionality to v6?

Reply:
Yes. This will be backported to ES 6xx as well